internal/clients/nightfall/nightfall_internal_test.go

```diff
@@ -5,8 +5,9 @@ import (
 	"testing"
 
 	"github.com/nightfallai/nightfall_cli/internal/clients/diffreviewer"
+
 	githublogger "github.com/nightfallai/nightfall_cli/internal/clients/logger/github_logger"
-	"github.com/nightfallai/nightfall_cli/internal/nightfallconfig"
+
```

Comment:
nit - do we need these line breaks? Or can we just put them in one "paragraph"

Reply:
yea we typically want them in a paragraph, but goland sometimes adds line breaks for new imports that are added